### ISSUE TITLE
Add "crows' feet" to auto-generated ERD

### ISSUE THREAD

**Issue by esd-org-uk** [openreferral/specification#456]
(I hope this is the right place to log this.)
[This is the excellent ERD](https://docs.openreferraluk.org/en/latest/erd.html) generated by ODSC software for the UK profile.

[This is the ERD generated by Porism](https://github.com/OpenReferralUK/uk-profile/blob/main/docs/assets/ERD.png) software from the tabular data package.

The ODSC one is generally better but it lacks "crows' feet" denoting the direction of one-to-many relationships. **Can they be added?**

**Comment by mrshll1001** [comment]
I agree that adding crow's feet would be great, but sadly I don't think this is possible using the current software package used to generate the ERDs :-( If we wanted to add crow's feet, we'd need to investigate alternatives or seek to build something new.

At the moment, ERDs generated by the docs are generated via the Python package [jts_erd](https://pypi.org/project/jts_erd/) and are actually also generated from the tabular data package. See [conf.py, L451&ndash;L516](https://github.com/openreferral/specification/blob/3.0/docs/conf.py#L451) for how this is done.

I've had an investigate into the [jts_erd docs](https://jts-erd.readthedocs.io/en/latest/) including the [examples](https://github.com/iburadempa/jts_erd/blob/master/examples/example1.py) and cannot find a way to provide crow's feet inside the package. We'd either need to fork it, or develop something else from scratch.

Unfortunately, nothing I know of can generate ERDs directly from JSON Schema.

If the core issue is that the documentation lacks clear descriptions of where particular schemas are embedded under others in a 1-to-many fashion, we can consider using the [compiled schemas](https://github.com/openreferral/specification/tree/3.0/schema/compiled) and pointing a [docson viewer](https://github.com/lbovet/docson) at them. This renders beautiful views of JSON Schemas, and ODSC maintains its [own fork](https://github.com/opendataServices/docson) adding features for use with specific standards. This would generate a view of the data which demonstrates the embedded nature of JSON, as well as highlight where there are arrays.
